### PR TITLE
New command: slcli order quote-delete

### PR DESCRIPTION
**Pull request by caberos**
[#issue1915](https://github.com/softlayer/softlayer-python/issues/1915)

image

### CODE REVIEW

File: SoftLayer/CLI/order/quote_delete.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+from SoftLayer.CLI.command import SLCommand as SLCommand
+from SoftLayer.CLI import environment
+from SoftLayer.managers import ordering
+
+
+@click.command(cls=SLCommand)
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """View a quote"""
```
Comment (edsonarios) [inline SoftLayer/CLI/order/quote_delete.py:14]: Maybe should be "Delete the quote of an order"

**Comment by allmightyspiff** [comment]
image

File: SoftLayer/CLI/order/quote_delete.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Delete the quote of an order"""
+
+    manager = ordering.OrderingManager(env.client)
+    result = manager.delete_quote(identifier)
+
+    if result:
+        env.fout("Quote id: {} was deleted.".format(identifier))
```
Comment (allmightyspiff) [inline SoftLayer/CLI/order/quote_delete.py:20]: I'd like to start using Python F strings going forward, as they read a little better.

```suggestion
        env.fout(f"Quote id: {identifier} was deleted.")
```

**Review by allmightyspiff — CHANGES_REQUESTED** [review]
Looks good, just a slight change to use  [F strings](https://peps.python.org/pep-0498/). Thanks

**Comment by ramkishor-ch** [comment]
Hi,

I have "tested the changes".
Please find the screenshot for reference.

Screen Shot: 1
Screenshot 2023-05-02 at 11 19 56 PM

Screen Shot: 2
Screenshot 2023-05-02 at 11 36 26 PM

Screen Shot: 3
Screenshot 2023-05-02 at 11 21 32 PM

Screen Shot: 4
Screenshot 2023-05-02 at 11 23 50 PM

Thanks,
Ramkishor Chaladi.

**Review by ramkishor-ch — APPROVED** [review]
Hi
I have "tested the changes".
Approving the Pull Request.
Thanks,
Ramkishor Chaladi.

**Review by allmightyspiff — APPROVED** [review]

**Pull request merged**